### PR TITLE
Add e2e test for different runtimes and allow them to be parallely runned 

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- add test for python37
- add abilit to run test for python and nodejs in the same time.
**Related issue(s)**

### CODE REVIEW

File: tests/function-controller/testsuite/teststep/configure.go [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+func (f ConfigreFunction) prettyPrint(obj interface{}) {
+	out, err := stringifyToYaml(obj)
+	if err != nil {
+		f.log.Warnf("error: %s", err)
+	} else {
+		f.log.Infof("%s", out)
+	}
+}
+
+func (f ConfigreFunction) LogResources() error {
```
Comment (aerfio) [inline tests/function-controller/testsuite/teststep/configure.go:154]: lowercase

**Review by aerfio — APPROVED** [review]
LGTM, most of the work was done by @dbadura

**Review by m00g3n — APPROVED** [review]

**Pull request merged**